minari/dataset/minari_storage.py

```diff
@@ -85,7 +87,21 @@ def apply(
             for ep_idx in episode_indices:
                 ep_group = file[f"episode_{ep_idx}"]
                 assert isinstance(ep_group, h5py.Group)
-                out.append(function(ep_group))
+                ep_dict = {
```

Comment:
on `apply` signature, change the signature of `function` to accept dictionary

Reply:
done